public/docs/_examples/displaying-data/dart/web/main.dart

```diff
@@ -0,0 +1,16 @@
+// #docregion
+import 'package:angular2/bootstrap.dart';
```

Comment:
That's a good point. But I agree with you. For the use case of teaching dev's angular2/Dart, I think this approach does feel easier to follow when looking at the examples directly.